orttraining/orttraining/models/bert/main.cc

```diff
@@ -554,7 +554,6 @@ void setup_training_params(BertParameters& params) {
   params.weights_not_to_train = {
       "position_01",            // Slice's dat input
       "op_min_ends_expand_10",  //op_min_ends_expand_10
```

Comment:
Are these still needed? I hope we could do this automatically

Reply:
It was previously needed to exclude these because there was no path to the loss node. This is now automatically detected and they are no longer needed.

Reply:
Awesome!